index.html

```diff
@@ -30,73 +30,276 @@ <h2 class="logo">
+                sign-ups required.</p>
+              <ul class="skills">
+                <li class="li-html">html</li>
+                <li class="li-css">css</li>
+                <li class="li-javascript">javascript</li>
+              </ul>
+            </div>
+
+            <div class="link-project"></div>
+            <a class="linktonic" href="#">See Project</a>
+          </div>
+        </div>
+      </article>
+
+    </section>
+
+    <section class="about">
+      <div class="container">
+        <div class="back-images-intro">
+
+          <h1>About<br>Myself</h1>
+          <p class="intro-text">I’m a software developer! I can help you
+            build a product, feature or website. Look
+            through some of my work and
+            experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <a href="#">
+            <h3 class="connectlink">LET´S CONNECT</h3>
+          </a>
+
+          <div class="socialmedia">
+            <ul class="icons">
+              <li>
+
+                <a href="#"><img src="./images/twitter.svg" alt="twitter"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/linkedin.svg" alt="linkedin"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/m.svg" alt="M"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/github.svg" alt="Git"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/vector.svg" alt="V"></a>
+              </li>
+            </ul>
+          </div>
+          <div class="link-project"></div>
+          <a class="linktonic" href="#">Get my resume</a>
+        </div>
+      </div>
+    </section>
+
+    <section class="lang-frame-skills">
+      <ul class="lang-frame-skills_languages">
+        <li>Languages</li>
+        <li><img src="./images/union.svg" class="arrow-down" alt="arrow down"></li>
+      </ul>
+      <ul class="lang-frame-skills_icons">
+        <li class="javascript">
+          <img class="skills-image" src="./images/js.svg" alt="JS" /><span>JavaScript</span>
+        </li>
+        <li class="html5">
+          <img class="skills-image" src="./images/html5.svg" alt="Html5" />HTML
+        </li>
+        <li class="css">
+          <img class="skills-image" src="./images/css3.svg" alt="CSS" />CSS
+        </li>
+      </ul>
+      <ul class="lang-frame-skills_languages lang-frame-skills_frameworks">
+        <li>Frameworks</li>
+        <li><img src="./images/arrowright.svg" class="arrow-right" alt="arrow right"></li>
+      </ul>
+      <ul class="lang-frame-skills_languages lang-frame-skills_skills">
+        <li>Skills</li>
+        <li><img src="./images/arrowright.svg" class="arrow-right" alt="arrow right"></li>
+      </ul>
     </section>
+    <section class="my-form">
```

Comment:
- [ ] Kindly remove this section since is not part of the milestone
![Capturecont](https://user-images.githubusercontent.com/81444338/171226010-1403eccd-4111-48d4-aa3e-b9f1725b77d7.PNG)